installer/scripts/omsadmin.sh

```diff
@@ -346,6 +362,28 @@ set_proxy_setting()
     fi
 }
 
+set_omsagent_proc_limit()
```

Comment:
Good point, when reviewing I only taken omsconfig and omsagent process related users into an account.
@lagalbra , can you look into Atchut's concern?